test/Dapr.SecretStore.ConfigurationExtension/DaprSecretStoreConfigurationProviderTest.cs

```diff
@@ -0,0 +1,91 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Dapr;
+using Autogenerated = Dapr.Client.Autogen.Grpc;
+using FluentAssertions;
+using Xunit;
+using Microsoft.Extensions.Configuration;
+using Dapr.Client;
+using Grpc.Net.Client;
+using System.Net;
+
+namespace Dapr.Microsoft.Extensions.Configuration.DaprSecretStore.Test
```

Comment:
Rename to Dapr.Configuration.Extensions.Test.

Reply:
Done